CHANGELOG.md

```diff
@@ -88,6 +88,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - bcopy: fix crash in bcopy when using certain cli arguments [PR #1211]
 
 ### Changed
+- docs: describe Debian installation based on add_bareos_repositories.sh [PR #1238]
```

Comment:
Wouldn't be make more sense to see that in Documentation section?

Reply:
Done.